modules/nixpkgs.nix

```diff
@@ -20,6 +20,9 @@
             (inputs'.nixpkgs or (throw "flake-parts: The flake does not have a `nixpkgs` input. Please add it, or set `perSystem._module.args.pkgs` yourself."))
             inputs'.nixpkgs.legacyPackages
         );
+        _module.args.pkgsets = lib.mkOptionDefault (
```

Comment:
Putting `mkOptionDefault` around the whole thing will cause the whole thing to disappear when the user (or a module) defines a pkgset of their own.
By moving it into the individual attrs, they become individualy overridable, but removing pkgsets will require a `mkForce` on the whole thing. This seems to be a better trade-off because removing isn't super useful unless someone has some radical ideas that I'm unaware of :)